tests/ci/run_posix_tests.sh

```diff
@@ -26,6 +26,9 @@ build_and_test -DOPENSSL_NO_ASM=1 -DCMAKE_BUILD_TYPE=Release
 echo "Testing building shared lib."
 build_and_test -DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release
 
+echo "Testing FIPS entropy source: Passive."
+build_and_test -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DUSE_FIPS_ENTROPY_SOURCE_PASSIVE=1
```

Comment:
Is this testing with passive entropy? From the CMake list I think you also need to turn on FIPS:
```suggestion
build_and_test -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DUSE_FIPS_ENTROPY_SOURCE_PASSIVE=1 -DFIPS=1
```

Reply:
Nice find

Reply:
Using the `fips_build_and_test` function instead.